index.html

```diff
@@ -32,6 +32,16 @@
           branch: "master"
         },
         localBiblio: {
+          "ID3-EMSG": {
+            title: "Timed Metadata in the Common Media Application Format (CMAF)",
```

Comment:
Please note that the title of the specification is changing, see https://github.com/AOMediaCodec/id3-emsg/issues/22

Reply:
Thanks, yes. I'll wait for the spec to change before updating this PR. It would be good also to have a specref entry (please!)